Resources/Prototypes/Entities/Objects/Consumable/Drinks/drinks_cups.yml

```diff
@@ -365,3 +365,31 @@
     - HEAD
     quickEquip: false
     sprite: Clothing/Head/Hats/party_water_cup.rsi
+
+- type: entity
+  parent: DrinkBaseCup
+  id: DrinkHotChamomile
```

Comment:
It might be an idea to create a new base prototypes for teas, since there are now multiple different types with only a few differences between them